src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -122,6 +122,7 @@ const DONATION_DONOR_TYPE_ENUM = Object.freeze({
 });
 const CREDIT_CARD_WIDGET_NAME = 'geFormWidgetTokenizeCard';
 const ACH_CONSENT_MESSAGE = 'true';
+const PAYMENT_METHOD_CREDIT_CARD = 'Credit Card';
```

Comment:
This constant is also defined in src/lwc/rd2EntryForm/rd2EntryForm.js and src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js (and also within the script in GE_TokenizeCard.page)... assuming these are equivalent defined values (corresponding to the picklist value in the Payment Method picklist, right?) is there some way to reduce the number of duplicated constants? 

Also assuming this is tied to the picklist value, is there a risk of that value getting translated?